projects/storefrontlib/src/shared/utils/validators/custom-form-validators.spec.ts

```diff
@@ -167,8 +167,20 @@ describe('FormValidationService', () => {
   });
 
   describe('Star rating validator', () => {
-    const invalidValues = [null, 'a', 0, 1000];
-    const validValues = [1, 2, 3, 4, 5];
```

Comment:
according to our discuss, let's revert that

Reply:
reverted